src/server/config.ini

```diff
@@ -94,6 +94,7 @@
   io_service_worker_count = 4
   ; how many connections can be established from one ip address to a server(both replica and meta), 0 means no threshold
   conn_threshold_per_ip = 0
+  enable_udp = false
```

Comment:
UDP is enabled before, the default value would better to be `true` to keep compatiblity, you can define it what ever you like in your own envs.